src/hooks/useStablecoinPrice.ts

```diff
@@ -62,7 +63,7 @@ export default function useUSDCPrice(currency?: Currency): Price<Currency, Token
 }
 
 export function useUSDCValue(currencyAmount: CurrencyAmount<Currency> | undefined | null) {
```

Comment:
should this function also not be `useStablecoinValue?`

Reply:
It should, but that's probably a lot of churn, so let's save it for a follow up.